pkg/utils/kubernetes/reduce/reduce.go

```diff
@@ -165,6 +166,22 @@ func ReduceHPAs(ctx context.Context, k8sClient client.Client, hpas []autoscaling
 	return nil
 }
 
+// +kubebuilder:rbac:groups=policy,resources=poddisruptionbudgets,verbs=delete
+
+// PDBFilterFunc filters a list of PodDisruptionBudgets.
+type PDBFilterFunc func(hpas []policyv1.PodDisruptionBudget) []policyv1.PodDisruptionBudget
```

Comment:
🙃 

```suggestion
type PDBFilterFunc func(pdbs []policyv1.PodDisruptionBudget) []policyv1.PodDisruptionBudget
```

We might as well mention in the type comment what return value can be used for (is this the slice of entities to remove or those are the ones that have been removed?)